src/components/node-list/styles/_row.scss

```diff
@@ -11,17 +16,24 @@
   display: flex;
   align-items: center;
 
-  // force gpu layers to avoid drawing lag on scroll
+  // Fixed height required for lazy list (see note above).
+  height: 37px;
```

Comment:
We might want to add a comment that it needs to match a hard-coded variable in `node-list-row.js` as well.

Reply:
Ah that was noted above this comment, either way I've now moved it directly to this line instead.